### PR TITLE
Feature/debug cice

### PR DESCRIPTION
**Pull request by DeniseWorthen**
Resolves issue #7. All regression test pass on hera: [https://github.com/DeniseWorthen/ufs-s2s-model/tree/feature/rtcicedebug/log](url)

### CODE REVIEW

File: lanl_cice/bld/Macros.Linux.NEMS.cheyenne [hunk 1]
```diff
@@ -4,16 +4,24 @@
 # For use with intel compiler with MPT 
 #==============================================================================
 
+DEBUG =Y
```
Comment (JessicaMeixner-NOAA) [inline lanl_cice/bld/Macros.Linux.NEMS.cheyenne]: Do you want to hard code this to Y?

**Review by JessicaMeixner-NOAA — APPROVED** [review]
Not sure if you want to hard code the debug=Y on Cheyenne, but looks good to me otherwise!

File: lanl_cice/bld/Macros.Linux.NEMS.cheyenne (hunk 1 above)
Comment (DeniseWorthen) [reply lanl_cice/bld/Macros.Linux.NEMS.cheyenne]: I removed the setting--thanks!

**Review by jiandewang — APPROVED** [review]
look good

**Pull request merged**